README.md

```diff
@@ -1,85 +1,79 @@
+| [Creating &amp; modifying **simple** pages](creating_modifying_simple_pages.md)      |
+| [Creating &amp; modifying **complex** pages](creating_modifying_complex_pages.md)    |
+| [Adding features in Python &ndash; advanced](adding_features_python.md)              |
+| [Adding meeting minutes](minutes.md)                                                 |
+| [Writing server READMEs](servers.md)                                                 |
+| [Development process and using git &ndash; beginner's guide](development_and_git.md) |
 
-### 'Regular' pages
-These can be found in `regular/`, and consist of a YAML header and a HTML body, which is inserted into the template `wrapper.handlebars`.
-
-#### YAML fields
- - **title**: put into the `<title>` element of the page.
-
-### News articles
-Written in [Markdown], with a YAML header:
- - **title**: the title of the article, put into the `<title>` and `<h1>` elements and shown in the list of news articles.
-
-The first paragraph of the article is shown as an excerpt in the news article list, Markdown requires a blank line (not just a single newline) to indicate a new paragraph.
-
-### Minutes
-For the moment, minutes are in PDF form and are put into `minutes/`. These are copied into the web directory, along with an index page which lists all the minutes in the directory. The filename format is `YYYY-MM-DD-meeting name.pdf`. Regular committee meetings are simply `YYYY-MM-DD-Minutes n.pdf`. The minutes page sorts these in ascending date order based on the filename, so omitting the date will cause this to fail.
-
-### Templates
-Most of the site is generated through the `wrapper.handlebars` template. This takes the content of the page, the title, plus some global values found in `templates/context.yaml`:
-
- - `servers`: controls the list of servers on the website banner
-    - `name`: name of this server (capitalised, hostname only)
-    - `href`: link to this server's README (typically just `http://<hostname>.hacksoc.org/`).
- - `nav`: links for navbar
-    - `text` and `href`: fairly self-explanatory
+## Running
+Preferred way to run is through the `flask` command. It can be executed with the venv (see [Installation](#installation)) activated by just running `flask` or otherwise at `venv/bin/flask`:
 
+### Starting a development server
+```
+flask run
+venv/bin/flask run
+```
+Pages are served directly from the Flask routes; you shouldn't need to restart the server when changes are made, but web pages will not automatically refresh. Open your browser to [`http://localhost:5000/`](http://localhost:5000/) to see the results.
 
-### Server READMEs
-Written in [Markdown]. Like regular pages, has a YAML header:
- - `hostname`: lowercase hostname of the server
- - `fqdn`: full domain of the server, typically `<hostname>.hacksoc.org`
- - `name`: purpose/subtitle of the server (eg "shell server")
+### Freezing to HTML
```

Comment:
It isn't really clear to me what this means? Maybe there's more detail in the proper docs, and this may be my "never written a proper website"-ness showing, but I don't know what freezing to html is, nor whether I need to do it if I want to change the website. A quick google search for "freezing html" gives me a bunch of different things so isn't especially helpful.